docs/OCP_CI_Tutorials/Scenarios/Reporting_Guide.md

```diff
@@ -0,0 +1,128 @@
+    - `open_test_template`: Template for the URL linking to a specific test.
+      - `url`: URL template. This should not change for our purposes.
+    - `results_url_template`: Template for the URL linking to specific test results.
+      - `url`: URL template. This should not change for our purposes.
+
+- `test_groups`: A list of Prow jobs to be able to use in your dashboard.
+  - `name`: Essentially an alias for the Prow job name. We use this name in the `test_group_name` key above. This value needs to be unique to not only this document, but other `test_groups` items in other config files.
+  - `gcs_prefix`: The Google Cloud Storage prefix path for the Prow results. Should always be `origin-ci-test/logs/` plus the name of the Prow job.
+  - `days_of_results`: How many days of results should be seen on TestGrid. I haven't seen this value go above 60 days, but I also haven't tried going any higher than that.
+
+### How to Create a New Dashboard
+
+Creating a new TestGrid dashboard is very simple. Before you get started you will need to [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository into your personal GitHub organization.
+
+
+Once you have your fork created, follow these instructions to create a new dashboard:
+
+1. Prepare Git:
+   1.  Clone a copy of your fork to your computer
+   2.  Create a new branch and name it whatever you'd like.
+2. In your new branch, create a new file in the `config/testgrids/openshift` directory. The name of the file should be the same name as your new dashboard
+   - **Example:** `config/testgrids/openshift/redhat-openshift-4.12-interop.yaml`
+
+> **IMPORTANT**:
+>
+> Do not use `-release-` at all in your file name or your dashboard name. This will break the OpenShift CI automation that generates their TestGrid dashboards. See this pull request for more information: [kubernetes/test-infra PR #28593](https://github.com/kubernetes/test-infra/pull/28593 )
+
+3. Write your configuration file.
+   - **Note:** Use the example in the [TestGrid Configuration Example and Description](#testgrid-configuration-example-and-description) section of this document.
+4. After you have pushed your changes to your fork, follow the [pull request process](#pull-request-process) to submit a PR and have your dashboard created.
+
+### How to Add a Job to a Dashboard
+
+If there is already a suitable dashboard created in TestGrid to hold your new Prow job, follow this guide to get it appended to the existing dashboard. Before you get started you will need to [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) the [kubernetes/test-infra](https://github.com/kubernetes/test-infra) repository into your personal GitHub organization.
+
+1. Prepare Git:
+   1.  Clone a copy of your fork to your computer
```

Comment:
```suggestion
   1.  Clone a copy of your fork to your computer.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)